gatsby-config.ts

```diff
@@ -1,8 +1,9 @@
-import type { GatsbyConfig } from "gatsby";
+import type { GatsbyConfig } from 'gatsby';
 
-require("dotenv").config({
-  path: `.env.${process.env.NODE_ENV}`,
-})
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config({
```

Comment:
unfortunately Gatsby does not support ESM in `gatsby-XYZ.ts` files yet :( So this must stay like this, thx for the eslint-disable